### PR TITLE
manifold/specs: create logspout daemonset forwarding logs to papertrail

### PR DESCRIPTION
**Pull request by mattatcha**

### CODE REVIEW

File: manifold/specs/logspout.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    metadata:
+      name: logspout
+      labels:
+        app: logspout
+    spec:
+      containers:
+      - image: gliderlabs/logspout:v3.1
+        imagePullPolicy: Always
+        name: logspout
+        args: ["syslog+tls://logs5.papertrailapp.com:31048"]
```
Comment (mattatcha) [inline manifold/specs/logspout.yaml:17]: Probably shouldn't hardcode the endpoint here.

Comment (progrium) [reply manifold/specs/logspout.yaml:17]: It's just this one place, pretty clear from filename where it might be. I say leave it.

**Comment by progrium** [comment]
These aren't automatically applied. Needs to also be added here https://github.com/gliderlabs/infra.gl/blob/master/manifold/cluster.tf#L53

Could probably make that load the dir, but would involve spinning up the stack. Maybe we can do that later when it's easier to spin up in a single account.

**Review by progrium — APPROVED** [review]

**Review by progrium — CHANGES_REQUESTED** [review]
Add to cluster.tf

**Review by progrium — APPROVED** [review]

**Pull request merged**